prow/oss/run-k8s-binary.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# This script is for running arbitrary Go binary from k8s/test-infra repo,
+# running this script when this script cannot run docker container, for example
+# when the Go binary need to authenticate to GCP.
+# Installing the binary directly from k8s/test-infra repo instead of add
+# k8s.io/test-infra in a sub go module inside current directoty, as we have seen
+# dependency problems when doing so in other repos.
+#
+# Usage:
+# $1       : The relative path for installing Go binary from k8s/test-infra
+# $2..n    : The arguments for running the Go binary
+#
+# Example:
+# - ./run-k8s-binary.sh prow/cmd/generic-autobumper --config=prow/oss/oss-autobump-config.yaml
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}/../..")" && pwd -P)"
+
+TMP_DIR="$(mktemp -d)"
+K8S_INFRA_DIR="${TMP_DIR}/test-infra"
+BINARY="${TMP_DIR}/binary"
+
+# Install the Go binary directly from k8s/test-infra
+git clone git@github.com:kubernetes/test-infra.git "$K8S_INFRA_DIR"
+cd "$K8S_INFRA_DIR"
+go build -o "${BINARY}" "$1"
+shift 1
```

Comment:
Nit: `shift` alone is the same as `shift 1`.

Reply:
done